CMakeLists.txt

```diff
@@ -357,6 +352,13 @@ if (NOT S2N_EXECINFO_AVAILABLE)
 endif()
 feature_probe_result(S2N_STACKTRACE ${S2N_STACKTRACE})
 
+if(TSAN)
+    target_compile_options(${PROJECT_NAME} PUBLIC -fsanitize=thread)
+    target_link_options(${PROJECT_NAME} PUBLIC -fsanitize=thread)
+    # For now, only enable real atomics for TSAN
+    feature_probe_result(S2N_ATOMIC_ENABLED ${S2N_ATOMIC_SUPPORTED})
```

Comment:
This will always happen without this line since we glob on the directory and try to compile the feature file. [Example](https://us-west-2.codebuild.aws.amazon.com/project/eyJlbmNyeXB0ZWREYXRhIjoidUpNb2QzNjZzQ3lrWm1UQ2xzdFZMN1ZGYWE5dVlSQ1ZKd1BnZzZ6aXN1dlZXTmFuZWdvNnR0SitPaFlIcUdjaDZ6RU9KemY2Mi9ybXVBajF0RXo4NmZ1Rnl4dkZrT241UUY4MTVQMkRuNTQ9IiwiaXZQYXJhbWV0ZXJTcGVjIjoiemFRUDgvZUMreVhER0RKaiIsIm1hdGVyaWFsU2V0U2VyaWFsIjoxfQ%3D%3D/build/cd8277ad-9f67-4072-b4d7-3e9c1d23a5fc) - Line 243

Reply:
Line 243 is S2N_ATOMIC_**SUPPORTED**. Here we're setting S2N_ATOMIC_**ENABLED**, which is what we check in the code, since for now we don't want to use the atomic builtins even if they're supported.

I might need better names 😅 Any suggestions?